app/layout.tsx

```diff
@@ -19,8 +20,10 @@ export default function RootLayout({
       <body>
         <PythonRuntimeProvider>
           <FridgeProvider>
+            <FeatureFlagProvider>
               <NavBar />
               {children}
+            </FeatureFlagProvider>
```

Comment:
please make `FeatureFlagsProvider` an ancestor of `FridgeProvider` if you are going to use the feature flags in the `FridgeProvider`

Reply:
Done